bin/queue

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Application;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\ContainerConfig;
+
+$autoload = $GLOBALS['_composer_autoload_path'] ?? './vendor/autoload.php';
+
+if (!file_exists($autoload)) {
+    throw new RuntimeException(
+        'Please run "composer install" in your project root to install the required dependencies.'
+    );
+}
+
+require_once $autoload;
+
+/** @var array $definitions */
+$definitions = require_once 'definitions.php';
```

Comment:
```suggestion
$definitions = require_once __DIR__ . 'definitions.php';
```

Reply:
If you use the absolute route, when you copy the definitions to the ROOT Directory of the app, not work.

Reply:
Not need copy definitions.

Reply:
If necessary, suppose that i have a different configuration than the predetermined, for example, i used monolog, when updating will lose my changes, that is fine, only when your configuration is equal to the predetermine